### PR TITLE
QA: PDP, Product Grid Item, Collection Links Module

### PR DESCRIPTION
**Pull request by chrismekim**
### Description
QA Items:

- Text Highlighting should invert colors (black highlight, white text)
- Product Description feels a little too wide to be naturally readable
- The typography relationship on the Collection card bugs my eyeballs out a bit
- Homepage Product Area
- Product Type on the PDP isn't vertically centered
- PDP Product title

### Type(s) of changes

- [x] Bug fix
- [x] Update to an existing feature

### Motivation for PR

See description
### How Has This Been Tested?

https://9hyczm9245opjoy1-52674822324.shopifypreview.com
pw: nunu

### CODE REVIEW

File: theme/snippets/collection-links.liquid
Comment (chrismekim) [inline theme/snippets/collection-links.liquid:9]: If there are links, loop through them and render "or" in front of the link if it's the last item in the for loop (and not a single link).

File: theme/snippets/news-feed.liquid
Comment (chrismekim) [inline theme/snippets/news-feed.liquid:14]: The "See more" link after the Homepage News Feed

(Thought I would make it editable since they have asked to edit text)

File: theme/styles/buttons.scss
Comment (chrismekim) [inline theme/styles/buttons.scss:81]: Waiting on Jake to give me the mobile font sizes so this will probably need adjustment

File: theme/styles/global.scss
Comment (chrismekim) [inline theme/styles/global.scss:114]: `.bulleted `replaces the need for these `.dots `

File: theme/styles/settings/_typography.scss
Comment (chrismekim) [inline theme/styles/settings/_typography.scss:176]: Makes highlight text white on black

File: theme/styles/snippets/collection-links.scss
Comment (chrismekim) [inline theme/styles/snippets/collection-links.scss:13]: Waiting on Jake to give me the mobile font sizes

File: theme/styles/snippets/product-info-bar.scss
Comment (chrismekim) [inline theme/styles/snippets/product-info-bar.scss:30]: "Cropping" the line height to make it more visually centered vertically. This doesn't change the actual line height of the text but visually looks more centered next to the ●

[Ticket](https://www.notion.so/garden3d/ba130e56c01f4d9284d8bbf6cf59ba0f?v=133f51dd370c40ad981fef24024fa8b6&p=1dfe2e0fecfe444ebbeff8fa4e2d86fe)

**Review by sepowitz — APPROVED** [review]
LGTM! Not sure if you are waiting on stuff from Jake be4 merging, but feel free to merge if ur ready

**Pull request merged**
